jekyll/_cci2/deployments.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: classic-docs
+title: "Deployments"
+short-title: "Deployments"
+categories: [configuring-jobs]
+order: 98
+---
+
+There are two standard ways to do deployment in 2.0: using a [deploy]({{ site.baseurl }}/2.0/configuration-reference/#deploy) step and triggering a deploy job.
+
+## Using a Deploy Step
+
+```YAML
+version: 2
+jobs:
+  build:
+    docker:
+      - image: my-image
+    working_directory: /tmp/my-project
+    steps:
+      - run: <do-some-stuff>
+      - deploy:
+          name: Maybe Deploy
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              <your-deploy-commands>
+            fi
+```
+
+The above is an example of configuring your job to do deployment by using a `deploy` step with some pseudo commands.
+
+The most important part to be stressed in the example is that we are checking whether the current branch is `master` branch.
+
+```
+command: |
+  if [ "${CIRCLE_BRANCH}" == "master" ]; then
+    ...
+  fi
+```
+
+Without this checking, `<your-deploy-commands>` will be executed every time this job is triggered.
+
+To learn more about `deploy` step, please checkout our [doc]({{ site.baseurl }}/2.0/configuration-reference/#deploy).
+
+## Triggering a Deploy Job
+
+```YAML
+version: 2
+jobs:
+  build:
+    docker:
+      - image: my-image
+    working_directory: /tmp/my-project
+    steps:
+      - run: <do-some-stuff>
+      - deploy:
+          name: Maybe Triggering Deploy Job
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+                curl -u ${CIRCLE_API_TOKEN}: \
+                     -d build_parameters[CIRCLE_JOB]=deploy \
+                     https://circleci.com/api/v1.1/project/<vcs-type>/<org>/<repo>/tree/master
```

Comment:
You mean the fact that we ask customers to use `curl` to invoke a new job? I feel it's ok until we have a workflow but do you have any suggestions?